setup.py

```diff
@@ -233,31 +251,27 @@ def read(fname):
                             'setproctitle',
                             'ntplib',
                             'msgpack-python',
-                            'pyzmq'], 
+                            'pyzmq'],
     'extras_require'     : {'autopilot' : ['github3.py']},
-    'tests_require'      : ['mock==2.0.0', 'pytest'],
+    'tests_require'      : ['pytest',
+                            'pylint',
+                            'flake8',
+                            'coverage',
+                            'mock==2.0.0.',
+                           ],
     'test_suite'         : '%s.tests' % name,
     'zip_safe'           : False,
-  # 'build_sphinx'       : {
-  #     'source-dir'     : 'docs/',
-  #     'build-dir'      : 'docs/build',
-  #     'all_files'      : 1,
-  # },
-  # 'upload_sphinx'      : {
-  #     'upload-dir'     : 'docs/build/html',
-  # },
-    # This copies the contents of the examples/ dir under
-    # sys.prefix/share/$name
-    # It needs the MANIFEST.in entries to work.
-  # 'data_files'         : makeDataFiles('share/%s/examples/' % name, 'examples'),
     'data_files'         : df,
+    'cmdclass'           : {'upload': RunTwine},
```

Comment:
Did not find what this is.

Reply:
calling `python setup.py upload` then pushes the module to pypi when this is defined, removing some manual steps in the release process.  Twine is the python tool which handles the pypi upload and some checks while doing so.